nwbwidgets/test/test_timeseries.py

```diff
@@ -11,3 +11,31 @@ def test_timeseries_widget():
                     rate=100.)
 
     traces_widget(ts)
+
+
+
+from pynwb import TimeSeries
+from nwbwidgets.utils.timeseries import get_timeseries_tt
+
+
+def test_get_timeseries_tt():
+    
+    data = list(range(100, 200, 10))
+    ts = TimeSeries(name='test_timeseries', data=data, unit='m', starting_time=0.0, rate=1.0)
+    
+    get_timeseries_tt(ts)
```

Comment:
For functions that have a numeric output, test that the output is correct with `np.testing.assert_array_equal`

Reply:
@Armin12  please address this